src/Security/SecurityConfigUpdater.php

```diff
@@ -61,18 +64,28 @@ public function updateForAuthenticator(string $yamlSource, string $firewallName,
 
         $firewall = $newData['security']['firewalls'][$firewallName];
 
-        if (!isset($firewall['guard'])) {
-            $firewall['guard'] = [];
-        }
+        if ($useSecurity51) {
+            if (!isset($firewall['custom_authenticator'])) {
+                $firewall['custom_authenticator'] = $authenticatorClass;
+            }
```

Comment:
Shouldn't this append the custom authenticator to the list of custom authenticators?

E.g. if there already is one of these 2:
```yaml
security:
  firewalls:
    main:
      # ...
      custom_authenticator: 'App\Security\FirstAuthenticator'
```
```yaml
security:
  firewalls:
    main:
      # ...
      custom_authenticator:
        - 'App\Security\FirstAuthenticator'
```

I think the result should be:
```yaml
security:
  firewalls:
    main:
      # ...
      custom_authenticator:
        - 'App\Security\FirstAuthenticator'
        - 'App\Security\NewMakerAuthenticator'
```

Or is this not how other make commands behave as well? (I'm not an experienced maker bundle user)

Reply:
I think Wouter is right. Should we (can't we) use `custom_authenticators`? So... we would need to check to see if either keys are configured?

Reply:
Yes, indeed. `custom_authenticators` and `custom_authenticator` are both allowed.

Reply:
done